### PR TITLE
SCC-3375: Always displaying the logged-in state (React 15)

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Resolves [SCC-3375](https://jira.nypl.org/browse/SCC-3375) for the React 15 version.

Vega does not currently support reading NYPL cookies. Therefore, when a patrons signs into the catalog, the NYPL Header will not be able to display "My Account" or the patron's name dynamically. This is a temporary (?) change that will always render the logged in state. This includes:

- Always rendering "My Account" in header dropdown button.
- Always rendering the link texts to "Go to..." the Catalog or Research Catalog. Previously, it was "Log in to..." when a patron was not logged in.
- Not rendering the patron's name or the logout link when they are logged in.

Most tests that are not relevant are removed.

### CODE REVIEW

File: src/components/Header/MobileHeader.js [hunk 1]
```diff
@@ -192,19 +192,16 @@ class MobileHeader extends React.Component {
...
-          className="loginIcon-loggedIn animated fadeIn"
-          ariaHidden
-          focusable={false}
-        />
-      );
-    }
+    const gaAction = 'MyAccount';
+    let icon = (
+      <LoginIconSolid
+        className="loginIcon-loggedIn"
```
Comment (nonword) [inline src/components/Header/MobileHeader.js:198]: This was the one slight functional difference I see with the 16 update - the removal of the "animated" and "fadeIn" classes. I assume that was intentional

**Review by nonword — APPROVED** [review]

File: src/components/Header/MobileHeader.js (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/Header/MobileHeader.js:198]: Good catch, thanks. Yes, the removal of those classes are intentional and I'll update the React 16 branch.

Comment (EdwinGuzman) [reply src/components/Header/MobileHeader.js:198]: Updated.

**Review by charmingduchess — APPROVED** [review]
Looks good!

**Pull request merged**